demos/src/scss/resized.scss

```diff
@@ -9,5 +9,5 @@ $o-grid-gutters: (default: 5px);
 @import 'common';
 
 .test body:after {
-	content: '$o-grid-layouts: S: 400px, M: 800px, L: 1000px, $o-grid-gutters: #{$o-grid-gutters}';
+	content: '$o-grid-layouts: S: 400px, M: 800px, L: 1000px, $o-grid-gutters: 5px';
```

Comment:
yeah, probably none of them should be hardcoded, but it's not that important. We can leave it as is. Feel free to merge